lib/datasource/terraform-provider/index.ts

```diff
@@ -13,14 +13,17 @@ import type {
 export class TerraformProviderDatasource extends Datasource {
   static readonly id = 'terraform-provider';
 
+  static readonly defaultRegistryUrls = [
+    'https://registry.terraform.io',
+    'https://releases.hashicorp.com',
+  ];
+
```

Comment:
This simplifies access, so we don't need a new instance for accessing static registry urls.